Syllabus.md

```diff
@@ -0,0 +1,116 @@
+3400 Civic Center Blvd
+215-746-8083
+bvoight@upenn.edu
+
+### Casey S. Greene, PhD
+Department of Systems Pharmacology and Translational Therapeutics
+10-131 Smilow Center for Translational Research (SCTR)
+3400 Civic Center Blvd
+215-573-2991
+csgreene@upenn.edu
+
+## Teaching assistants
+
+* [Katie Siewert](mailto:ksiewert@mail.med.upenn.edu)
+* TBD
+
+### Guest proctors:
+
+* [Greg Grant](mailto:ggrant543@gmail.com)
+
+## Course web Site
+
+We will use [SageMathCloud](https://cloud.sagemath.com/) for in class exercises
+and homework. Completed assignments will be collected through this system.
+
+## Course description
+
+This course provides broad overview of bioinformatics and computational biology
+as applied to biomedical research. A primary objective of the course is to
+enable students to integrate modern bioinformatics tools into their research
+activities. Course material is aimed to address biological questions using
+computational approaches and the analysis of data.
+
+A basic primer in programming and operating in a UNIX environment will be
+presented, and students will also be introduced to Python, R, and tools for
+reproducible research. This course emphasizes direct, hands-on experience with
+applications to current biological research problems. The course is **NOT
+INTENDED** for computer science students who want to learn about biologically
+motivated algorithmic problems; GCB/CIS/BIOL536 and GCB537 is more appropriate.
+
+Areas include DNA sequence alignment, genetic variation and analysis, motif
+discovery, study design for high-throughput sequencing, RNA and gene expression,
+single gene and whole-genome analysis, machine learning, and topics in systems
+biology. The relevant principles underlying methods used for analysis in these
+areas will be introduced and discussed at a level appropriate for biologists
+without a background in computer science.
+
+The course will assume a solid knowledge of modern biology. An advanced
+undergraduate course such as BIOL421 or a graduate course in biology such as
+BIOL526 (Experimental Principles in Cell and Molecular Biology), BIOL527
+(Advanced Molecular Biology and Genetics), BIOL528 (Advanced Molecular
+Genetics), BIOL540 (Genetic Systems), or equivalent, is a prerequisite.
+
+## Equipment prerequisite
+
+All students are required to bring a laptop to class for in class activities.
+TAs will provide help with the material, but students should be computer-capable
+with their own laptop, and should be willing/capable to download and install
+free software from the Internet.
+
+## Grading
+
+### Composition of grades
+
+Grades are composed of:
+
+* the course project (50% total, 25% written proposal,
+25% oral presentation)
+* homework assignments (30%)
+* in-class labs assignments (10%)
+* class participation (10%).
+
+For the homework grade, we will drop scores from the 3 lowest scoring
+assignments when determining the final grade.
+
+### Late grading policy:
+
```

Comment:
Last year, there was interest in receiving a faster turnaround on assignments than this 1-week policy allowed (at least - I got questions around when assignments would be returned). What do you think about converting it to a 24-hour policy which would enable us to turn around Monday assignments - at least for the week-long modules - while the week is still ongoing. Alternatively we could provide no credit for late assignments, which would allow a turnaround by the next class period. I think there are three good reasons to do this:

* We are dropping the 3 lowest scoring assignments, so this is not as harsh as it may at first seem.
* It allows the students who are staying up to date with assignments to benefit from rapid feedback.
* Making up missed assignments by a deadline increases the workload on the students who missed the deadline right as they are in danger of falling behind.